web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -144,42 +162,55 @@ export class StylesheetManager {
         // with embedded ttf or woff.  svg mostly works so is a better initial
         // choice on the Android browser.
         if(svg != '') {
-          s0.push("url('"+svg+"') format('svg')");
+          source = "url('"+svg+"') format('svg')";
         }
 
         if(woff != '') {
-          s0.push("url('"+woff+"') format('woff')");
+          source = "url('"+woff+"') format('woff')";
```

Comment:
Ideally we should be supporting woff2 as well but that probably should be a separate PR